internal/db/copy.go

```diff
@@ -42,3 +40,73 @@ func CopyAllObjects(ctx context.Context, tx pgx.Tx, source int64, target int64)
 
 	return nil
 }
+
+func CloneToProject(ctx context.Context, tx pgx.Tx, source int64, target int64, vrange VersionRange) (*int64, error) {
+	withHash := true
+
+	objectQuery := &pb.ObjectQuery{
+		Path:        "",
+		IsPrefix:    true,
+		WithContent: false,
+		WithHash:    &withHash,
```

Comment:
Is there a better approach than to add this to protobuf object (it changes the grpc signatures)?

Reply:
Ah yea that could work, but knowing that protobuf defaults to false for missing properties maybe keeping it here is ok.